vault/raft.go

```diff
@@ -1239,9 +1239,9 @@ func (c *Core) raftLeaderInfo(leaderInfo *raft.LeaderJoinInfo, disco *discover.D
 	return ret, nil
 }
 
-// getRaftBackend returns the RaftBackend from the HA or physical backend,
+// GetRaftBackend returns the RaftBackend from the HA or physical backend,
 // in that order of preference, or nil if not of type RaftBackend.
-func (c *Core) getRaftBackend() *raft.RaftBackend {
+func (c *Core) GetRaftBackend() *raft.RaftBackend {
```

Comment:
Can you remind me why you're exporting this now?  I checkout out your branch and looked for references to GetRaftBackend, and they all seemed to be from package `vault`, meaning it could stay private, based on this PR at least.  Is there an ENT version of the PR where that's not the case?

Reply:
The enterprise PR test accesses the backend from `rafttests` package: https://github.com/hashicorp/vault-enterprise/pull/3815/files#diff-887dfbdeec2a88de4e2792bcbe13a8e3b609b3962ad136bf9487667ed66d0062R588

Reply:
I threw an alternate idea in there: https://github.com/hashicorp/vault-enterprise/pull/3815/files#r1156055469

I don't insist, I think the current approach is acceptable too.  My suggestion is motivated by a desire to minimize how much non-core code talks directly to the raft backend instead of going through APIs.

Reply:
No that's fair. I don't mind changing it. Gimme a few mins ;)